cli/src/config-schema.json

```
@@ -168,6 +168,16 @@
                     "type": "string",
                     "enum": ["none", "watchman"],
                     "description": "Whether to use an external filesystem monitor, useful for large repos"
+                },
+                "watchman": {
+                    "type": "object",
+                    "properties": {
+                      "set_up_trigger": {
+                        "type": "boolean",
+                        "default": false,
+                        "description": "Whether to set up a watchman trigger to enable automatic snapshotting on file system modifications"
```

```suggestion
                        "description": "Whether to use triggers to monitor for changes in the background."
```

suggestion: I think the comment you used elsewhere is more succinct and more helpful in this case.